packages/docgen/src/is-symbol-private.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+const getSymbolTagsByName = require( './get-symbol-tags-by-name' );
+
+/**
+ * Returns true if, given a symbol object, the symbol is considered private and
```

Comment:
Nitpick (_wish [suggestions](https://help.github.com/en/articles/reviewing-proposed-changes-in-a-pull-request) had multi-line support_):

```suggestion
 * Returns true if, given a symbol object, it contains a @private tag, or false otherwise.
```

the fact that this is a reusable function makes it valuable for omitting symbols from formatted output or any other use case.

Reply:
> the fact that this is a reusable function makes it valuable for omitting symbols from formatted output or any other use case.

Good call! Updated in fc5933f.